### PR TITLE
Update to 2.3.8

### PR DESCRIPTION
**Pull request by lapineige**
## Problem
- Wallabag is not up-to-date.

## Solution
- Update the source

Minor release, mainly bug fixing: https://github.com/wallabag/wallabag/releases/tag/2.3.8
Last one before 2.4 (which is likely to require more work).

## PR Status
- [x] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : Kay0u
- [x] **Approval (LGTM)** : Kay0u
- [x] **Approval (LGTM)** : Maniack C
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/wallabag2_ynh%20PR71/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by lapineige** [comment]
@nicofrand please give it a try :)
Warning this will not include this #69 until we merge it into testing (see #70).

**Comment by nicofrand** [comment]
Works fine!

File: manifest.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "en": "A self hostable read-it-later app",
         "fr": "Une application de lecture-plus-tard auto-hébergeable"
     },
-    "version": "2.3.7-1~ynh2",
+    "version": "2.3.8-1~ynh2",
```
Comment (kay0u) [inline manifest.json]: I think the version number should be "2.3.8~ynh1"
The upstream version is 2.3.8 and not 2.3.8-1 (or I miss something?)
And the "yuno version number": ~ynhX is reset each time the upstream version change.

Comment (lapineige) [reply manifest.json]: Oh, I changed the version number without even paying attention that we were in ~ynh2 ^^

**Comment by nicofrand** [comment]
For the `~ynh1` you mean?

> version: version of the package builded from the upstream version number and an incremental number for each change in the package without upstream change. Example "1.0.1~ynh7". Must be a string.

https://yunohost.org/#/packaging_apps_manifest_en

**Comment by kay0u** [comment]
> For the `~ynh1` you mean?
> 
> > version: version of the package builded from the upstream version number and an incremental number for each change in the package without upstream change. Example "1.0.1~ynh7". Must be a string.
> 
> https://yunohost.org/#/packaging_apps_manifest_en

I'm aware of this :-), I mean, the "-1" in "2.3.8-1"

**Comment by lapineige** [comment]
I don't see a reason why, but @JimboJoe might know.

**Comment by maniackcrudelis** [comment]
@lapineige, next time, please to not forget to update the link to the CI. So we can have the result of the automatic tests.

**Comment by lapineige** [comment]
Well, I tried, but somehow I did not understand how to do it, it didn't work.

**Comment by lapineige** [comment]
Let's merge in 3 days then ?

**Comment by kay0u** [comment]
Hup!

**Comment by JimboJoe** [comment]
@lapineige I don't see any reason for the "-1" to be there...

**Comment by lapineige** [comment]
Fixed.
Don't bother to review it again. It will be merged in 2 days anyway.

**Pull request merged**